decidim-templates/config/locales/en.yml

```
@@ -42,6 +42,15 @@ en:
         name: Template
     templates:
       admin:
+        block_user_templates:
+          form:
+            save: Save
+            template_title: Template information
+          index:
+            confirm_delete: Are you sure you want to delete this template?
+            title: Block User Messages
```

Let's change the capital letters here please 😄 

```suggestion
            title: Block user messages
```